botblocker.go

```
@@ -36,52 +38,75 @@ type BotBlocker struct {
+
+	b.prefixMutex.Lock()
+	b.prefixBlocklist = prefixBlockList
+	b.prefixAllowlist = prefixAllowList
+	b.prefixMutex.Unlock()
+	return nil
+}
+func fetchPrefixes(urls []string) ([]netip.Prefix, error) {
+	prefixList := make([]netip.Prefix, 0)
+
+	for _, url := range urls {
 		resp, err := http.Get(url)
```

golangci-lint (2.13.2)

[error] response body must be closed

(bodyclose)